float64 pitch
float64 yaw_angle
float64 lat
float64 lon
float64 alt